openlibrary/macros/LoanStatus.html

```diff
@@ -87,6 +87,9 @@
   $else:
     <a href="$work.key" class="cta-btn cta-btn--missing">$_('Not in Library')</a>
 
+$else:
```

Comment:
I've tested (on works, editions, lists pages) that this doesn't double up `Not in Library` buttons. Without this line -- test by disabling it -- you will see items on https://dev.openlibrary.org/people/mekBot/lists/OL143580L/Patrick_Collison's_Bookshelf not show up with any CTA

Reply:
Which ones show up with "Not in Library"? Why?